source/_docs/scripts/conditions.markdown

```diff
@@ -239,10 +240,31 @@ Note that if only `before` key is used, the condition will be `true` *from midni
 If only `after` key is used, the condition will be `true` from the specified time *until midnight*.
 Time condition windows can span across the midnight threshold if **both** `after` and `before` keys are used. In the example above, the condition window is from 3pm to 2am.
 
-<div class='note tip'>
+<div class='note warning'>
+As of release 0.109, the weekday property will be deprecated. Replace it with weekday condition.
```

Comment:
We don't keep change log warnings/notes in the documentation. Those go into the breaking changes section of the release notes. Please remove it.

Reply:
removed